Sources/NIO/EventLoop.swift

```diff
@@ -1366,16 +938,19 @@ private final class ScheduledTask {
 }
 
 extension ScheduledTask: CustomStringConvertible {
+    @usableFromInline
     var description: String {
-        return "ScheduledTask(readyTime: \(self.readyTime))"
+        return "ScheduledTask(readyTime: \(self._readyTime))"
     }
 }
 
 extension ScheduledTask: Comparable {
+    @inlinable
     static func < (lhs: ScheduledTask, rhs: ScheduledTask) -> Bool {
-        return lhs.readyTime < rhs.readyTime
+        return lhs._readyTime < rhs._readyTime
     }
 
+    @inlinable
```

Comment:
Similarly, why does this need to be `inlinable`?

Reply:
removed, was  semi-automatic replacement